conda-forge.yml

```diff
@@ -1,7 +1,12 @@
-build_platform: {osx_arm64: osx_64}
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
+  osx_arm64: osx_64
```

Comment:
I don't understand why the migration switches aarch/ppc to cross-compilation here?

Reply:
https://github.com/regro/cf-scripts/issues/1518